client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -237,7 +241,10 @@ export const List = React.createClass( {
 	},
 
 	addDomainButton() {
-		if ( ! config.isEnabled( 'upgrades/domain-search' ) ) {
+		if (
+			! config.isEnabled( 'upgrades/domain-search' ) ||
+			this.props.selectedSite.jetpack
```

Comment:
We disabled the add domain button on the assumption that we didn't want to allow adding a domain to Jetpack sites. This is something that I'd like some input on from store and maybe @beaulebens to see if it's possible. And if it's not possible, should it be?